scout/build/hpo.py

```diff
@@ -41,4 +42,6 @@ def build_hpo_term(hpo_info):
         children=list(hpo_info.get("children", set())),
     )
 
-    return hpo_obj
+    dict_hpo_term: Dict = dict(hpo_obj)
+    dict_hpo_term["_id"] = hpo_obj.hpo_id
```

Comment:
Assigning _id to the dictionary to be saved in the database is a bit weird, I am doing outside the pydantic class because I've found out that pydantic ignores fields that start with underscore(https://github.com/pydantic/pydantic/issues/2105)

Basically what happened was that the object didn't have an id, and when it was saved it got an Object ID assigned by default by pymongo instead of a string one like this _id:HP006545